### PR TITLE
PR0030-266 reporter implementation

### PR DESCRIPTION
**Pull request by vadim-fob**

### CODE REVIEW

File: src/main/groovy/io/cify/runner/utils/PluginExtensionManager.groovy [hunk 1]
```diff
@@ -143,6 +144,13 @@ class PluginExtensionManager {
     }
 
     /**
+     * Set test run id
+     */
+    private void setRunId(){
+        project.cify.runId = getValue('runId')
```
Comment (markokonsaFob) [inline src/main/groovy/io/cify/runner/utils/PluginExtensionManager.groovy:150]: If run id is empty we should generate some random id to run

File: src/main/groovy/io/cify/runner/CifyPluginExtension.groovy [hunk 2]
```diff
@@ -28,6 +28,11 @@ class CifyPluginExtension {
     String cucumberPlugins = "pretty,json:build/cify/reports/json/"
 
     /**
+     * Test run id. Default is empty string
```
Comment (markokonsaFob) [inline src/main/groovy/io/cify/runner/CifyPluginExtension.groovy:31]: Default shouldn't be empty string

File: src/test/groovy/io/cify/runner/utils/PluginExtensionManagerTest.groovy [hunk 3]
```diff
@@ -74,6 +74,10 @@ class PluginExtensionManagerTest extends GroovyTestCase {
         fileWithMode = testProjectDir.newFile("fileWithMode.json")
     }
 
+    void testDefaultRunId() {
+        assert extension.runId == defaultExtension.runId
+    }
+
     void testGetDefaultFeatureDir() {
```
Comment (markokonsaFob) [inline src/test/groovy/io/cify/runner/utils/PluginExtensionManagerTest.groovy:81]: Add test that run id is not empty

**Review by markokonsaFob — CHANGES_REQUESTED** [review]
Please fix those places

**Review by markokonsaFob — APPROVED** [review]
